src/main/java/frc/team2412/robot/util/motorcontroller/MotorController.java

```diff
@@ -28,66 +28,35 @@ public IdleMode getREV() {
 	}
 
 	public enum MotorControlMode {
-		POSITION {
-			@Override
-			public ControlMode getCTRE() {
-				return ControlMode.Position;
-			}
-
-			@Override
-			public ControlType getREV() {
-				return ControlType.kPosition;
-			}
-		},
-		VELOCITY {
-			@Override
-			public ControlMode getCTRE() {
-				return ControlMode.Velocity;
-			}
-
-			@Override
-			public ControlType getREV() {
-				return ControlType.kVelocity;
-			}
-		},
-		PERCENT {
-			@Override
-			public ControlMode getCTRE() {
-				return ControlMode.PercentOutput;
-			}
-
-			@Override
-			public ControlType getREV() {
-				return ControlType.kDutyCycle;
-			}
-		},
-		VOLTAGE {
-			@Override
-			public ControlMode getCTRE() {
-				return ControlMode.Velocity;
-			}
-
-			@Override
-			public ControlType getREV() {
-				return ControlType.kVelocity;
-			}
-		};
+		POSITION(ControlMode.Position, ControlType.kPosition),
+		VELOCITY(ControlMode.Velocity, ControlType.kVelocity),
+		VOLTAGE(
+				null,
+				ControlType.kVoltage), // CTRE does not have a voltage control mode, we just set it with
+		// motor.setVoltage()
```

Comment:
Personal preference, but moving line comments to separate lines tends to produce better formatting in my opinion (here and elsewhere, including in `DriveCommand` and `DrivebaseSubsystem`). Feel free to disregard.